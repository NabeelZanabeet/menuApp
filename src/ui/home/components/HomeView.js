import * as React from 'react';
import { View, Text } from 'react-native';
import styles from './HomeView.styles';

const HomeView = ({}) => {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>Home screen</Text>
    </View>
  );
};

export default HomeView;
