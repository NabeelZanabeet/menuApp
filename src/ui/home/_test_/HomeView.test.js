import 'react-native';
import React from 'react';
import deepRenderer from 'react-test-renderer';
import HomeView from '../components/HomeView';

describe('HomeView component', () => {
  test('renders correctly', () => {
    const tree = deepRenderer.create(<HomeView />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
