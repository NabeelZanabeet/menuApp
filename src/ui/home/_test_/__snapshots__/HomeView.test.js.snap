// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../src/assets/mkonnektLogo.png",
      }
    }
    style={
      Object {
        "height": 90,
        "marginBottom": 72,
        "width": 562.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0077be",
        "borderRadius": 8,
        "height": 40,
        "justifyContent": "center",
        "marginBottom": 16,
        "opacity": 1,
        "width": 337.5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 17,
        }
      }
    >
      Items
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0077be",
        "borderRadius": 8,
        "height": 40,
        "justifyContent": "center",
        "marginBottom": 16,
        "opacity": 1,
        "width": 337.5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 17,
        }
      }
    >
      Categories
    </Text>
  </View>
</View>
`;
